tests/gitgitgadget.test.ts

```diff
@@ -194,22 +194,18 @@ test("generate tag/notes from a Pull Request", async () => {
 
     const pullRequestURL = "https://github.com/gitgitgadget/git/pull/1";
     // tslint:disable-next-line:max-line-length
-    const description = `My first Pull Request!
-
-This Pull Request contains some really important changes that I would love to${
-        ""} have included in [git.git](https://github.com/git/git).
-
-Cc: Some Body <somebody@example.com>
-`;
-    const match2 = description.match(/^([^]+)\n\n([^]+)$/);
-    expect(match2).toBeTruthy();
+    const description = `This Pull Request contains some really important changes that I would love to${
```

Comment:
This is no longer the `description` but the `body`, and it should be accompanied by the `title`.

Also, this change removes the very `Cc:` footer that the first commit in this PR claims was previously untested.

It would be better to keep that footer and add a _separate_ submitter to the `getFromNotes()` call below.

Reply:
I will add a new variable `title` and rename `description` to `body`. I will add another `Cc` line.

> Also, this change removes the very Cc: footer that the first commit in this PR claims was previously untested.

Since GitGitGadget always adds author to `Cc:` I decided that here the line is the auto-appended author. So I merely preserved the old test scenario. But yes, this can be seen in a different way.